### DISCUSSION TITLE
Disable the "Automatically skip the beatmap intro" feature for now

### DISCUSSION THREAD

**Discussion by soopax** [ppy/osu#24040] [Q&A] [answered]
As shown in https://github.com/ppy/osu/issues/23583, the "Automatically skip the beatmap intro" setting in multiplayer has been broken, probably ever since it was added. It causes huge delays and various problems in the rooms that are using it. Looks to me that https://github.com/ppy/osu/pull/23627 is still a work-in-progress, so I think it would be a good idea to dim/disable the setting until it's fixed. Alternatively at the very least the setting's text could be changed to something like "Automatically skip the beatmap intro (beta)" or "Automatically skip the beatmap intro (broken)", for the time being.

**Comment by peppy** [comment, marked as answer]
Please wait for a fix instead.

A delay caused by it is rate, and at most it's the same length as the intro.
